content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,117 @@ title: Configure the Datadog Tracing Library
+**Description**: Automatically apply specified HTTP headers as span tags. If a custom tag name is not specified, the tag key defaults to `http.request.headers.<normalized-header-name>` for request headers and `http.response.headers.<normalized-header-name>` for response headers.
+
+`DD_TRACE_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean <br>
+**Description**: Enables or disables the sending of traces to the trace agent.
+
+`DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP`
+: **Default**: <br>
+    ```
+    (?i)(?:(?:"|%22)?)(?:(?:old[-_]?|new[-_]?)?p(?:ass)?w(?:or)?d(?:1|2)?|pass(?:[-_]?phrase)?|secret|(?:api[-_]?|private[-_]?|public[-_]?|access[-_]?|secret[-_]?|app(?:lication)?[-_]?)key(?:[-_]?id)?|token|consumer[-_]?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)(?:(?:\s|%20)*(?:=|%3D)[^&]+|(?:"|%22)(?:\s|%20)*(?::|%3A)(?:\s|%20)*(?:"|%22)(?:%2[^2]|%[^2]|[^"%])+(?:"|%22))|(?:bearer(?:\s|%20)+[a-z0-9._\-]+|token(?::|%3A)[a-z0-9]{13}|gh[opsu]_[0-9a-zA-Z]{36}|ey[I-L](?:[\w=-]|%3D)+\.ey[I-L](?:[\w=-]|%3D)+(?:\.(?:[\w.+/=-]|%3D|%2F|%2B)+)?|-{5}BEGIN(?:[a-z\s]|%20)+PRIVATE(?:\s|%20)KEY-{5}[^\-]+-{5}END(?:[a-z\s]|%20)+PRIVATE(?:\s|%20)KEY(?:-{5})?(?:\n|%0A)?|(?:ssh-(?:rsa|dss)|ecdsa-[a-z0-9]+-[a-z0-9]+)(?:\s|%20|%09)+(?:[a-z0-9/.+]|%2F|%5C|%2B){100,}(?:=|%3D)*(?:(?:\s|%20|%09)+[a-z0-9._-]+)?)
+    ```
+: **Supported Input**: A regex string <br>
+: **Description**: Applies a regex to redact sensitive data from query strings on incoming HTTP requests. Matches are replaced with `<redacted>`. If an empty string is passed, no obfuscation occurs. The resulting value is reported in the `http.url` tag.
+
+### Diagnostics
+
+`DD_TRACE_LOG_DIRECTORY`
+: **Default**: Varies by SDK, environment, and runtime. [Read more here][2] <br>
+**Supported Input**: A valid directory path that exists on the system<br>
+**Not Supported In**: Java, Node.js, Ruby, Python SDKs<br>
+**Description**: Specifies the directory where tracer log files should be routed. If the directory does not exist, the SDK falls back to its default diagnostic logging method.
+
+### Agent
+
+`DD_TRACE_AGENT_URL`
+: **Default**: `http://localhost:8126` <br>
+**Supported Input**: A string representing an HTTP or UDS url <br>
+**Description**: The URL for connecting the tracer to the Datadog agent. Valid URL schemas include `http://` and `unix://` (UNIX Domain Sockets). This value takes precedence over `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set.
+
+### Unified Service Tagging
+
+`DD_VERSION`
+: **Default**: `null`, with an exception for the Node.js SDK, which can automatically retrieve the version number from package.json, if available<br>
+**Supported Input**: A string representing an application version <br>
+**Description**: Adds a version tag to all spans that have the same service name as the global service name (aka `DD_SERVICE`). Spans with a different service name do not get this tag.
```

Comment:
This may be confusing to customers as they ask "why would it be different?". We can explain that by linking to the [inferred services](https://docs.datadoghq.com/tracing/services/inferred_services/?tab=agentv7551) page

```suggestion
**Description**: Adds a version tag to all spans except for spans coming from inferred services.
```